CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ if(BUILD_GDASBUNDLE)
   ecbuild_bundle( PROJECT atlas GIT "https://github.com/ecmwf/atlas.git" TAG 0.29.0 )
 
 # External (required) observation operators
+  option("BUNDLE_SKIP_CRTM" "Don't build CRTM" "OFF") # Don't build crtm unless user passes -DBUNDLE_SKIP_CRTM=OFF
```

Comment:
This works but it seems odd that setting an option `OFF` turns on   an action (ie, `OFF` = execute ecbuild_bundle for crtm).   Would something like `BUNDLE_BUILD_CRTM` or `BUILD_CRTM_BUNDLE` with `YES` as triggering ecbuild be clearer?

Reply:
1000% agree... This is the ecbuild syntax though for some reason. It knows to skip X if you set BUNDLE_SKIP_X where X is the name of a PROJECT within the bundle

Reply:
I was unaware of this behavior.    We need to work with what we have.

Reply:
I always wondered if I was the only one confused by this!